internal/pipe/defaults/defaults.go

```diff
@@ -36,6 +36,7 @@ func (Pipe) Run(ctx *context.Context) error {
 		}
 
 		ctx.Config.GiteaURLs.Download = strings.ReplaceAll(apiURL, "/api/v1", "")
+		ctx.Config.GiteaURLs.Download = strings.TrimSuffix(apiURL, "/")
```

Comment:
ah, I think this should be

```suggestion
		ctx.Config.GiteaURLs.Download = strings.TrimSuffix(ctx.Config.GiteaURLs.Download, "/")
```